### ISSUE TITLE
[JENKINS-71495] WCAG Operable:2.1 - Keyboard operation and navigation - Result 3. There is a loss of focus in important places.

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-14#7025]
<p><em>This problem has been noticed at the following important points:</em> </p>
<ul>
	<li><em>Operating the menu > new page content is not directly focused</em></li>
	<li><em>Selection of radio buttons > focus is set to the top of the page</em></li>
</ul>

<p>After selecting a menu item from the navigation menu and then loading the new pages, the keyboard focus is shifted to the top of the page. Screen reader users are not notified of the new content and must navigate again from the top of the page to the actual main content of the page.</p>

<p>Since the main content area is not marked as a region either, the navigation menu here cannot be skipped.</p>

<p>Even users with motor disabilities cannot navigate efficiently, since the keyboard focus is shifted to the top of the page after selecting a menu item.</p>

<p><b>Suggested fix:</b><b>{</b>}</p>

<p>For efficient navigation, the focus should be on the new content for dynamically displayed content using Java Script, so that blind and motorized users can see it directly.</p>

<p>It makes sense to set the focus directly on the first heading in the new content, so that non-sighted keyboard users in particular receive an orientation reference to the new content. To do this, this heading must also be made focusable with tabindex="-1" so that the focus can be set via JavaScript.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=cristinap">cristinap</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-71495" target="_blank">WCAG Operable:2.1 - Keyboard operation and navigation - Result 3. There is a loss of focus in important places.</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p><em>This problem has been noticed at the following important points:</em> </p>
<ul>
	<li><em>Operating the menu &gt; new page content is not directly focused</em></li>
	<li><em>Selection of radio buttons &gt; focus is set to the top of the page</em></li>
</ul>

<p>After selecting a menu item from the navigation menu and then loading the new pages, the keyboard focus is shifted to the top of the page. Screen reader users are not notified of the new content and must navigate again from the top of the page to the actual main content of the page.</p>

<p>Since the main content area is not marked as a region either, the navigation menu here cannot be skipped.</p>

<p>Even users with motor disabilities cannot navigate efficiently, since the keyboard focus is shifted to the top of the page after selecting a menu item.</p>

<p><b>Suggested fix:</b><b>{</b>}</p>

<p>For efficient navigation, the focus should be on the new content for dynamically displayed content using Java Script, so that blind and motorized users can see it directly.</p>

<p>It makes sense to set the focus directly on the first heading in the new content, so that non-sighted keyboard users in particular receive an orientation reference to the new content. To do this, this heading must also be made focusable with tabindex="-1" so that the focus can be set via JavaScript.</p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<i>[Epic: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-14/issues?q=in%3Atitle%20JENKINS-71153%20label%3Aepic">JENKINS-71153</a>]</i>
